packages/protocol-http/src/Field.ts

```diff
@@ -1,18 +1,18 @@
 import { FieldPosition } from "./FieldPosition";
 
-export type FieldOptions = {
+export type FieldOptions = { 
```

Comment:
I think prettier was not run on this file, which introduces space and removes semicolon.
The commit might have been created with `--no-verify`

Can you explicitly run prettier on this file? If using VSCode, it should run automatically if you open the file and click save.

Reply:
There were a few other files with the same issue, I've pushed the update.